games/Call 911.js

```diff
@@ -0,0 +1,695 @@
+..DDD...C...DDD.
+..D.....C.....D.
+........C.......
+........C.......
+........C.......
+........C.......
+........C.......
+........C.......
+........C.......
+........C.......`],
+  [selectLeft, bitmap`
+................
+................
+................
+................
+................
+................
+1111111111111111
+1000000000000000
+1000000000000000
+1000000000000000
+1000000000000000
+1000000000000000
+1000000000000000
+1000000000000000
+1000000000000000
+1111111111111111`],
+  [selectRight, bitmap`
+................
+................
+................
+................
+................
+................
+1111111111111111
+0000000000000001
+0000000000000001
+0000000000000001
+0000000000000001
+0000000000000001
+0000000000000001
+0000000000000001
+0000000000000001
+1111111111111111`],
+  [selectMiddle, bitmap`
+................
+................
+................
+................
+................
+................
+1111111111111111
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+1111111111111111`],
+  /*[bg, bitmap`
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000`]*/
+);
+
+setSolids([firetruck,house1,house2,pine,coconut,bushes]);
+
+let level = 0;
+const levels = [
+  map`
+.........
+.........
+.........
+.........
+.........`, // main menu
+  map`
+p.........
+.e.fkfeif.
+.k.h.i.f..
+.e.e.f...h
+.fif.e.e..
+..h....kf.
+f...ie.f..
+ekfkhk...f
+.fhe.f.e..
+.......k.w`,
+  map`
+.....p....
+eifkehfhe.
+..........
+bekifhehie
+..........
+eifkehhfh.
+..........
+.ehfiehfke
+......r...
+fiefkehifw`,
+  map`
+p.........
+efhfehfeh.
+hkhkhfkkf.
+heiiiiiih.
+eiififhie.
+hiiihiiih.
+wkhkekfhe.
+.effhkkfk.
+.fekkkeef.
+.......m..`,
+  map`
+p.........
+eke.kfi.fe
+f.k.e.e.h.
+e.fmk.k...
+f.kei.fhf.
+b.........
+.fhhefkeif
+.eiewr....
+.kfekfhe..
+..........`,
+  map`
+p.........
+k.k.k.k.k.
+kbkbkbkbkb
+kkkkkkkkkk
+ifefeffefh
+ifehfeffeh
+iffefehefh
+iefhefefeh
+ieeeffhhhh
+hhhhhhhr.w`,
+  map`
+rh........
+.fe.h.ehfk
+.hi.fe....
+.fh.khfkm.
+..........
+ehf.iekfe.
+....hf....
+.heif..k..
+......ifeh
+ihkfwhkp.b`,
+  map`
+..........
+.b..b...b.
+......b...
+..........
+...b...b..
+b.........
+.....p..b.
+..........
+.b.....hhh
+.....bmhrw`,
+  map`
+pi.......w
+.er....h..
+.hikheif.h
+...e...e..
+.i.h.f...b
+.e...h.e..
+...e.k.fif
+ih.h.i....
+he.fhfhki.
+kim.......`,
+  map`
+h..ib...fh
+e.kfhef...
+h.e...i.k.
+f...kke.fi
+h...fpf.hm
+..h.f.i.e.
+.ef.....i.
+..h..f....
+k.kiehfeh.
+r......wf.`,
+  map`
+ph...f....
+.f.e..mk.f
+.h.h.hei.h
+...k.fwh.e
+hkhf.h.hfi
+b....e....
+f.hehf.h.f
+h......e.e
+i.fhe.fi..
+m...ir...e`,
+  map`
+.........
+.........
+.........
+.........
+.........`, // Win screen
```

Comment:
If you meant the last 3 levels as per the last 3 maps, then last map was not intended it was a mistake, kept it blank for future levels, and 2nd last is a win screen, Yes the win screen is all white but will display text.

Reply:
Can you now check and clarify which levels are facing issue?

Reply:
![image](https://user-images.githubusercontent.com/88567622/196540230-dc34aaa2-4d84-4f61-8f4e-451d9d000f12.png)
![image](https://user-images.githubusercontent.com/88567622/196540368-a2283f38-1243-4f20-b195-15f99042c7dc.png)
![image](https://user-images.githubusercontent.com/88567622/196540455-88970438-2585-4bee-acf4-0c164da7c1a1.png)

Reply:
maybe i'm playing badly

Reply:
![IMG_20221019_022717](https://user-images.githubusercontent.com/24205008/196544884-813968b1-089b-4381-bd6f-6cd0ec5d5ad4.jpg)
In this one, we have to fill the fire truck two times so that enough water is present to extinguish the fire.


![IMG_20221019_022912](https://user-images.githubusercontent.com/24205008/196545162-5d7140bd-48af-45d6-94b6-4e8e71df3c70.jpg)

In this one follow the path shown, and don't collect the water which is present ( that's a trap :P  )

![IMG_20221019_023227](https://user-images.githubusercontent.com/24205008/196545398-594b5322-fc7a-47fd-9827-41706952da5e.jpg)

In this one collect the water and follow the path as shown.


Do lemme know if you need any more infos.

Reply:
Ok!